lib/plugins/create/templates/plugin/index.js

```diff
@@ -0,0 +1,36 @@
+'use strict';
+
+// https://github.com/serverless/serverless/blob/master/docs/04-extending-serverless/01-creating-plugins.md
+class ServerlessPlugin {
+  constructor(serverless, options) {
+    this.serverless = serverless;
+    this.options = options;
+
+    this.commands = {
+      deploy: {
+        lifecycleEvents: [
+          'functions',
+        ],
+        options: {
+          function: {
+            usage:
+              'Specify the function you want to deploy '
+              + '(e.g. "--function myFunction" or "-f myFunction")',
+            required: true,
+            shortcut: 'f',
+          },
+        },
+      },
+    };
+
+    this.hooks = {
+      'deploy:functions': this.deployFunction.bind(this),
+    };
+  }
+
+  deployFunction() {
```

Comment:
And I would create separate functions for hello, world, beforeHello and afterWorld message and print messages for each one of them so users have a good understanding what they can do